.github/stable/core.txt

```diff
@@ -34,7 +34,7 @@ numpy==1.26.4
 osqp==0.6.5
 packaging==24.0
 pathspec==0.12.1
-PennyLane_Lightning==0.36.0
+PennyLane_Lightning==0.35.1
```

Comment:
kinda.. I stopped using the latest lightning in #5440 but accidentally modified the real deps branch instead of a test one. but this should be the way going forward